src/EventGrid/EventGrid/Properties/AssemblyInfo.cs

```diff
@@ -12,5 +12,5 @@
 
 [assembly: ComVisible(false)]
 
-[assembly: AssemblyVersion("1.2.3")]
-[assembly: AssemblyFileVersion("1.2.3")]
+[assembly: AssemblyVersion("2.0.0")]
```

Comment:
please revert this version back to 1.2.3

Reply:
Sure.. fixed. thanks